pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -501,6 +503,22 @@ func mutateContainerResource(c *corev1.Container, override *v1beta1.OverrideSpec
 			}
 		}
 	}
+	var found = false
+	for _, logLevel := range override.LogLevel {
+		if logLevel.ContainerName == c.Name {
+			c.Args = append(c.Args, fmt.Sprintf("--v=%d", logLevel.LogLevel))
+			found = true
+		}
+	}
+
+	if !found {
+		if c.Name == "git-sync" {
```

Comment:
This string should be a constant. I believe there is one already.

Reply:
done. updated to use `reconcilermanager.GitSync` constant